yolact.py

```diff
@@ -396,18 +44,31 @@ class Yolact(nn.Module):
         - pred_aspect_ratios: A list of lists of aspect ratios with len(selected_layers) (see PredictionModule)
     """
 
-    def __init__(self):
+    def __init__(self, 
+                 config_name="yolact_base_config",
```

Comment:
Ideally, you wouldn't have to modify `config.py` to use YOLACT as a pip package. Thus, I think this should accept either the name of the config or a config object (i.e., `Union[str, Config]`). This would require a similar change in `set_cfg` too.

Reply:
> accept either the name of the config or a config object (i.e., Union[str, Config]).  

isn't that what's there now? Name of the config. I can change to the `(str, Config)` but that seems the same to me. I was thinking about ommiting the `set_cfg()` and expecting the user to import/create the config they want and just pass it here directly. So `from data.config import yolact_base; net = Yolact(yolact_base)` ?